pyvista/core/grid.py

```diff
@@ -190,6 +190,10 @@ def _from_arrays(
         # Must at least be an x array
         if check_duplicates:
             raise_has_duplicates(x)
+
+        # edges are shown as triangles if x is not floating point
+        if not isinstance(x.dtype, np.floating):
```

Comment:
```suggestion
        if not np.issubdtype(x.dtype, np.floating):
```

Reply:
For whatever reason this has messed up the windows tests...